.github/data/sheriff-order

```diff
@@ -0,0 +1,14 @@
+220509
```

Comment:
I would suggest to move this constant to the beginning of the shell script. It will allow us to have only the names of potential sheriffs in this list.

Reply:
sure